### DISCUSSION TITLE
Will this work with Orgs

### DISCUSSION THREAD

**Discussion by not-a-ethan** [anuraghazra/github-readme-stats#1565] [Q&A] [answered]
Will the README stats work with organizations?

**Comment by rickstaa** [comment, marked as answer]
@ethan-master-coding Thanks a lot for your interest in this project! The current `master` branch does not work with organizations. 

It, however, is the most requested feature on this repo (see https://github.com/anuraghazra/github-readme-stats/issues/1). There is already a PR open for this feature https://github.com/anuraghazra/github-readme-stats/pull/1122. We are trying to merge this PR into the `master` branch.

For now, if you want to use the `github-readme-stats` with organizations, you can use the PR mentioned above. See https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-981419174 for an explanation of how this is done.
